easybuild/tools/github.py

```diff
@@ -896,7 +898,10 @@ def check_github():
         else:
             check_res = "OK"
     elif github_user:
-        check_res = "FAIL (unexpected exception: %s)" % push_err
+        if LooseVersion(git.__version__) < LooseVersion('1.0'):
+            check_res = "FAIL (outdated gitpython module, try updating first)"
```

Comment:
can you clarify 'outdated' in the error, i.e. mention what the minimal version is? and also mention the what version was found

something like

```python
check_res = "FAIL (GitPython version %s is too old, should be version %s or newer)" % (git.__version__, req_gp_ver)
```